src/main/java/edu/bu/metcs673/trackr/common/CommonDataUtil.java

```diff
@@ -0,0 +1,45 @@
+package edu.bu.metcs673.trackr.common;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+import edu.bu.metcs673.trackr.bankaccount.BankAccount;
+import edu.bu.metcs673.trackr.bankaccount.BankAccountService;
+import edu.bu.metcs673.trackr.user.TrackrUser;
+import edu.bu.metcs673.trackr.user.TrackrUserService;
+
+@Component
+public class CommonDataUtil {
```

Comment:
I have commented about this already. It's a base class that is meant to be extended not an util class which is broad.